i3pyblocks/blocks/dbus.py

```diff
@@ -247,22 +247,33 @@ def __init__(
             **kwargs,
         )
         self.format = format
+        self.metadata: Dict[str, str] = {
```

Comment:
I think this is unnecessary.

```suggestion
        self.metadata = {
```

Reply:
You might be right. I am new to type annotations in python so I just slapped it onto everything.